websites/C/Custom Status/dist/metadata.json

```diff
@@ -8,12 +9,12 @@
     "en": "Custom Status is a way to set custom statuses using PreMiD. You can customize everything but using custom images. You have to visit [this page](https://eggsy.xyz/projects/premid/custom-status) to start using!",
     "tr": "Custom Status, Discord'un kendi özelliği gibi isteğinize göre ayarlanabilen bir oynuyor kısmıdır. Bu servis ile PreMiD kullanarak istediğiniz resmi seçme olanağı dışında bir RPC ile yapabileceğiniz her şeyi yapabilirsiniz. Servisin çalışması için servisi ekleyip, [buradaki](https://eggsy.xyz/projects/premid/custom-status) sayfaya gitmelisiniz."
   },
-  "url": "eggsy.xyz",
-  "regExp": "eggsy[.]xyz[/]projects[/]premid[/]custom-status",
-  "version": "1.2.5",
-  "logo": "https://premid.is-inside.me/PsLr519L.PNG",
-  "thumbnail": "https://premid.is-inside.me/rWy3ggow.png",
-  "color": "#212121",
+  "url": ["eggsy.xyz", "eggsy-tailwind.netlify.app"],
+  "regExp": "[eggsy[.]xyz[/]projects[/]premid[/]custom-status|eggsy-tailwind[.]netlify[.]app[/]projects[/]premid[/]custom-status",
+  "version": "1.5.0",
```

Comment:
dont think you mean to put the `[` at the begin of the regex
and 1.2.5 to 1.5.0 at once? just go to 1.3.0

Reply:
but it's a big update :((

Reply:
If you change only metadata it should be 1.2.6 xD

Reply:
:p